tests/config_test.go

```diff
@@ -37,15 +39,15 @@ import (
 	"kubevirt.io/kubevirt/pkg/config"
 	"kubevirt.io/kubevirt/tests"
 	"kubevirt.io/kubevirt/tests/console"
-	cd "kubevirt.io/kubevirt/tests/containerdisk"
+	"kubevirt.io/kubevirt/tests/libvmi"
 )
 
 var _ = Describe("[Serial][rfe_id:899][crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-compute]Config", func() {
 
 	var virtClient kubecli.KubevirtClient
 
 	var CheckIsoVolumeSizes = func(vmi *v1.VirtualMachineInstance) {
-		pod := tests.GetRunningPodByVirtualMachineInstance(vmi, vmi.Namespace)
+		pod := tests.GetRunningPodByVirtualMachineInstance(vmi, util.NamespaceTestDefault)
```

Comment:
Why do we need this? Before the function could cope with different namespaces.

Reply:
@xpivarc Thanks! that is a left-over from some thought process which is not relevant now. Fixed.